src/lib/rmt/maria_db/current_root_password_dialog.rb

```diff
@@ -73,7 +73,7 @@ def ok_handler
 
   def root_password_valid?(password)
     RMT::Utils.run_command(
-      "echo 'show databases;' | mysql -u root -p%1 2>/dev/null",
+      "echo 'show databases;' | mysql -u root --password=\"%1\" 2>/dev/null",
```

Comment:
Can't you just single quotes instead of the escaped double quotes? Since all is surrounded by `""`, Ruby string interpolation should still work, right? Same for the other lines.